test/python/simulator/test_qasm_simulator.py

```diff
@@ -69,6 +70,59 @@ def test_qasm_simulator(circuit: QuantumCircuit, backend: QasmSimulatorBackend,
         assert abs(target[key] - counts[key]) < threshold
 
 
+def test_qasm_simulator_support_parametrized_gates(backend: QasmSimulatorBackend, shots: int):
+    """Test backend's adequate support of parametrized gates"""
+
+    theta_a = Parameter("theta_a")
+    theta_b = Parameter("theta_b")
+    theta_c = Parameter("theta_c")
+    circuit_1 = QuantumCircuit(2)
+    circuit_2 = QuantumCircuit(2)
+    circuit_1.ry(theta_a, 0)
+    circuit_1.rx(theta_b, 1)
+    circuit_2.rx(theta_c, 0)
+
+    # Test backend raises the right type of errors
+    with pytest.raises(AssertionError) as exc_info:
+        backend.run([circuit_1, circuit_2], [[np.pi / 2, np.pi / 2]], shots=shots).result()
```

Comment:
```suggestion
        backend.run([circuit_1, circuit_2], [[np.pi / 2, np.pi / 2]])
```
Can be a little simpler